e2e/playwright/secrets.ts

```diff
@@ -1,28 +1,5 @@
-import { readFileSync } from 'fs'
-
 const secrets: Record<string, string> = {}
-const secretsPath = './e2e/playwright/playwright-secrets.env'
-try {
-  const file = readFileSync(secretsPath, 'utf8')
-  file
-    .split('\n')
-    .filter((line) => line && line.length > 1)
-    .forEach((line) => {
-      // Allow line comments.
-      if (line.trimStart().startsWith('#')) return
-      const [key, value] = line.split('=')
-      // prefer env vars over secrets file
-      secrets[key] = process.env[key] || (value as any).replaceAll('"', '')
-    })
-} catch (error: unknown) {
-  void error
-  // probably running in CI
-  console.warn(
-    `Error reading ${secretsPath}; environment variables will be used`
-  )
-}
-secrets.token = secrets.token || process.env.token || ''
-secrets.snapshottoken = secrets.snapshottoken || process.env.snapshottoken || ''
+secrets.token = process.env.token || ''
```

Comment:
Well hang on, if we're in control of this then we can just use `VITE_KC_DEV_TOKEN` no?

Reply:
I believe we can do that yes

Reply:
Feel free to push to this branch I should just to something else

Reply:
Yeah, let's merge this as-is.

Reply:
Mmmh .local doesn't work well locally for this

Reply:
Had to add this dotenv line here for things to work with `.env.development.local`... I don't like that

Reply:
Same. I would personally like to go _all in_ on environment variables, making `process.env` (or language equivalent) the "interface" between our machines and the various runtimes. That ensures calls like `FOOBAR=my-override-value thing.sh` will always work. For another PR though!

Reply:
Alright I tested this on macos and windows